src/utils/throwOnUndefinedProperty/test.ts

```diff
@@ -5,5 +5,6 @@ test('throws an error when trying to access undefined property', () => {
   const protectedObj = throwOnUndefinedProperty(obj);
 
   expect(protectedObj.foo).toBe('bar');
+  // @ts-ignore
```

Comment:
Let's add to this comment explainer that TypeScript is catching the very error we're seeking to avoid throwing.

Reply:
done